.github/workflows/build-triton-wheel.yml

```diff
@@ -9,11 +9,13 @@ on:
       - .github/workflows/build-triton-wheel.yml
       - .github/scripts/build_triton_wheel.py
       - .github/ci_commit_pins/triton.txt
+      - .ci/docker/ci_commit_pins/triton.txt
```

Comment:
Is one triton.txt sufficient? Can you please test it?

Reply:
I think `.github/ci_commit_pins/triton.txt` can be removed.  We just need the source file in `.ci/docker/ci_commit_pins/triton.txt`.  I keep the original file `.github/ci_commit_pins/triton.txt` as a softlink here mainly to avoid causing a surprise when people expect it to be there and see that it isn't.

What do you think about removing `.github/ci_commit_pins/triton.txt` latter in the subsequent PR to clean up the CI?  My plan is to roll this out in 2 steps:

* This PR to add triton to Docker
* Another PR to clean up install_triton throughout the CI and may be also removing `.github/ci_commit_pins/triton.txt`

Reply:
Sounds good to me.